tests/integration/standard/test_metadata.py

```diff
@@ -1103,12 +1103,13 @@ def test_legacy_tables(self):
 
 CREATE TABLE legacy.composite_comp_no_col (
     key blob,
-    column1 'org.apache.cassandra.db.marshal.DynamicCompositeType(org.apache.cassandra.db.marshal.TimeUUIDType, org.apache.cassandra.db.marshal.BytesType, org.apache.cassandra.db.marshal.UTF8Type)',
```

Comment:
I don't know about the C* stakeholders since I am not familiar with how Thrift was or is still used, we'd have to ask @iamaleksey, but I do know that a dynamic composite type is implemented in CQL with clustering columns, read the "composites" section of [this blog](http://www.datastax.com/dev/blog/thrift-to-cql3).